api/go.mod

```diff
@@ -14,7 +12,7 @@ require (
 	github.com/gorilla/handlers v1.5.2
 	github.com/gorilla/mux v1.8.1
 	github.com/gorilla/websocket v1.5.3
-	github.com/h2non/filetype v1.1.3
+	github.com/hosom/gomagic v0.0.0-20160718182707-cbc00aac97a4
```

Comment:
`github.com/vimeo/go-magic/magic` exposes raw c functions, which `github.com/hosom/gomagic` wraps a little to align common pattern of Go functions.

`github.com/gabriel-vasile/mimetype` is a pure go library and I don't think it could catch up with the latest formats as `libmagic` is.